sql/src/main/java/io/crate/execution/engine/collect/stats/JobsLogs.java

```diff
@@ -145,7 +160,10 @@ public void operationStarted(int operationId, UUID jobId, String name) {
     }
 
     public Iterable<SysMetricsTableInfo.ClassifiedHist> metrics() {
-        return singletonList(new SysMetricsTableInfo.ClassifiedHist(histogram.copy(), "total"));
+        return histograms.entrySet()
+            .stream()
+            .map(e -> new SysMetricsTableInfo.ClassifiedHist(e.getValue().copy(), e.getKey()))
+            .collect(Collectors.toList());
```

Comment:
Please avoid `.collect(Collectors.toList())` if only a `Iterable` is returned. It's unnecessary memory usage and causes more allocations. Can use `::iterator` instead.